guides/keras_cv/cut_mix_mix_up_and_rand_augment.py

```diff
@@ -24,9 +24,11 @@
 """
 ## Imports & setup
 
-This tutorial requires you to have KerasCV installed:
+This tutorial requires you to have KerasCV installed. It also requires installing
+the TensorFlow nightly at this time.
 
 ```shell
+pip install tf-nightly
```

Comment:
you might want to put "pip install -U tf-nightly", which will force the update the dependency packages like keras-nightly